provider/provider.go

```diff
@@ -10,11 +10,24 @@ import (
 	"sync"
 
 	"github.com/ChainSafe/fil-secondary-retrieval-markets/shared"
+	"github.com/filecoin-project/specs-actors/actors/abi"
 	logging "github.com/ipfs/go-log/v2"
 )
 
 var log = logging.Logger("provider")
 
+// DefaultPricePerByte is the charge per byte retrieved if the miner does
+// not specifically set it
+var DefaultPricePerByte = abi.NewTokenAmount(2)
+
+// DefaultPaymentInterval is the baseline interval, set to 1Mb
+// if the miner does not explicitly set it otherwise
+var DefaultPaymentInterval = uint64(1 << 20)
+
+// DefaultPaymentIntervalIncrease is the amount interval increases on each payment,
+// set to to 1Mb if the miner does not explicitly set it otherwise
+var DefaultPaymentIntervalIncrease = uint64(1 << 20)
+
```

Comment:
I think it would be better to reference these directly, they could change https://github.com/filecoin-project/go-fil-markets/blob/caae3fda662396b73137ab44c8ee08d125cc570b/retrievalmarket/impl/provider.go#L73-L81

Reply:
attempting to import `"github.com/filecoin-project/go-fil-markets/retrievalmarket/impl"` causes what looks like cgo issues:
```
# pkg-config --cflags  -- /home/elizabeth/go/pkg/mod/github.com/filecoin-project/filecoin-ffi@v0.0.0-20200326153646-e899cc1dd072/generated/../filcrypto.pc /home/elizabeth/go/pkg/mod/github.com/filecoin-project/filecoin-ffi@v0.0.0-20200326153646-e899cc1dd072/generated/../filcrypto.pc /home/elizabeth/go/pkg/mod/github.com/filecoin-project/filecoin-ffi@v0.0.0-20200326153646-e899cc1dd072/generated/../filcrypto.pc /home/elizabeth/go/pkg/mod/github.com/filecoin-project/filecoin-ffi@v0.0.0-20200326153646-e899cc1dd072/generated/../filcrypto.pc
Failed to open '/home/elizabeth/go/pkg/mod/github.com/filecoin-project/filecoin-ffi@v0.0.0-20200326153646-e899cc1dd072/generated/../filcrypto.pc': No such file or directory
No package '/home/elizabeth/go/pkg/mod/github.com/filecoin-project/filecoin-ffi@v0.0.0-20200326153646-e899cc1dd072/generated/../filcrypto.pc' found
Failed to open '/home/elizabeth/go/pkg/mod/github.com/filecoin-project/filecoin-ffi@v0.0.0-20200326153646-e899cc1dd072/generated/../filcrypto.pc': No such file or directory
No package '/home/elizabeth/go/pkg/mod/github.com/filecoin-project/filecoin-ffi@v0.0.0-20200326153646-e899cc1dd072/generated/../filcrypto.pc' found
Failed to open '/home/elizabeth/go/pkg/mod/github.com/filecoin-project/filecoin-ffi@v0.0.0-20200326153646-e899cc1dd072/generated/../filcrypto.pc': No such file or directory
No package '/home/elizabeth/go/pkg/mod/github.com/filecoin-project/filecoin-ffi@v0.0.0-20200326153646-e899cc1dd072/generated/../filcrypto.pc' found
Failed to open '/home/elizabeth/go/pkg/mod/github.com/filecoin-project/filecoin-ffi@v0.0.0-20200326153646-e899cc1dd072/generated/../filcrypto.pc': No such file or directory
No package '/home/elizabeth/go/pkg/mod/github.com/filecoin-project/filecoin-ffi@v0.0.0-20200326153646-e899cc1dd072/generated/../filcrypto.pc' found
pkg-config: exit status 1
```

Reply:
Not ideal but probably better than adding the submodule dep :+1:

Fwiw https://github.com/filecoin-project/filecoin-ffi#go-get